src/main/java/com/emc/mongoose/load/step/client/LoadStepClientBase.java

```diff
@@ -54,9 +47,15 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
+import static com.emc.mongoose.Constants.KEY_CLASS_NAME;
+import static com.emc.mongoose.Constants.KEY_STEP_ID;
+import static com.emc.mongoose.config.ConfigUtil.flatten;
+import static org.apache.logging.log4j.CloseableThreadContext.Instance;
+import static org.apache.logging.log4j.CloseableThreadContext.put;
+
 public abstract class LoadStepClientBase
-extends LoadStepBase
-implements LoadStepClient {
+	extends LoadStepBase
```

Comment:
is it possible to configure the IDE for the project's code style?

Reply:
I tried many times, but it does not work out at all( and I can’t even manually remove this indent.

Reply:
ok